menuflow/example-config.yaml

```diff
@@ -1,13 +1,17 @@
 menuflow:
-  database: postgresql://menuflow:secretfoo@postgres/menuflow
-  database_opts:
-      min_size: 1
-      max_size: 10
+    database: postgresql://menuflow:secretfoo@postgres/menuflow
+    database_opts:
+        min_size: 1
+        max_size: 10
 
-  # User to be ignored
-  users_ignore:
-  - '@whatsappbot:example.com'
+    # Ignore messages from this users, it is a list of regex
+    ignore_user_messages:
+        - "@whatsappbot:example.com"
+        - "@admin:example.com"
 
+    ignore_invitations_from:
+        - "@whatsappbot:example.com"
+        - "@admin:example.com"
```

Comment:
Create a section called ignore and inside create the lists `messages_from` and `invitations_from`